components/tidb_query/src/rpn_expr/impl_math.rs

```diff
@@ -1,7 +1,10 @@
 use num::traits::Pow;
+use std::str::FromStr;
```

Comment:
This line is unused and won't pass lint I guess.

Reply:
I use it for Decimal::from_str.

Reply:
https://internal.pingcap.net/idc-jenkins/blue/organizations/jenkins/tikv_ghpr_test/detail/tikv_ghpr_test/13979/pipeline/8

It won't pass lint check (since only tests used and you need to put it in tests).